jobs/kibana/templates/bin/pre-start.erb

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if ! [[ $(ls -1 /var/vcap/packages/kibana/plugins | wc -l) == 0 ]]; then
```

Comment:
wait, what?

Reply:
Looks like this is checking to see if there are any files in this directory and if there are, remove them all; seems overly verbose, like if this has to happen, wouldn't the `rm -rf` statement be enough since it's operating on `*` in that directory?

Reply:
Don't shoot the messenger. I just merged the upstream.